ledger/sandbox-on-x/src/test/app/resources/logback.xml

```diff
@@ -1,31 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file was not needed, since the one from the main lib was already in resources and this one was no different.